### PR TITLE
mrc-215

### PR DESCRIPTION
**Pull request by hillalex**
The GitHub oAuth app id and secret need to be configured and passed into the web container. Also fixes a bug I found with the css generation logic - the volume was being mounted one directory level too high.

### CODE REVIEW

File: orderly_web/start.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def web_container(cfg, docker_client):
     image = str(cfg.images["web"])
     mounts = [docker.types.Mount("/orderly", cfg.volumes["orderly"])]
     if cfg.sass_variables is not None:
-        mounts.append(docker.types.Mount("/static/public", cfg.volumes["css"]))
+        mounts.append(docker.types.Mount("/static/public/css", cfg.volumes["css"]))
```
Comment (hillalex) [inline orderly_web/start.py]: Unrelated bug fix - found when testing locally

File: test/test_integration.py [hunk 2]
```diff
@@ -106,14 +106,18 @@ def test_start_with_custom_styles():
...
-        assert css_volume['Destination'] == "/static/public"
+        assert css_volume['Destination'] == "/static/public/css"
 
         # check that the style files have been compiled with the custom vars
         web_container = cfg.get_container("web")
         style = string_from_container(web_container,
                                       "/static/public/css/style.css")
         assert "/* Example custom config */" in style
 
+        # check that js files are there also
```
Comment (hillalex) [inline test/test_integration.py]: Unrelated bug that I found - js files were being overwritten by the mounted style volume.

File: test/test_integration.py [hunk 3]
```diff
@@ -189,6 +193,8 @@ def test_vault_ssl():
         cl.write("secret/ssl/certificate", value=cert)
         cl.write("secret/ssl/key", value=key)
         cl.write("secret/db/password", value="s3cret")
+        cl.write("secret/github/id", value="ghid")
+        cl.write("secret/github/secret", value="ghs3cret")
```
Comment (richfitz) [inline test/test_integration.py]: Can we check that these actually make it through into the app configuration?  Something like https://github.com/vimc/orderly-web-deploy/blob/fcd1f6ab26572e100b760bbd3663fda84344c8fb/test/test_integration.py#L62-L64
